site/content/en/docs/Concepts/services.md

```diff
@@ -82,13 +82,13 @@ The image section contains just a few parameters, the location of the Dockerfile
 
 #### HTTP
 
-The HTTP section is unique to the Load Balanced Web Service type. When a request comes to the load balancer, traffic will be fowraded to this service if the path matches '/' - meaning any traffic will be forwarded to this service. You could update this so that only traffic to the _front-end_ path would be routed to this service by updating the path to be `path: 'front-end'`.
+The HTTP section is unique to the Load Balanced Web Service type. When a request comes to the load balancer, traffic will be forwarded to this service if the path matches '/' - meaning any traffic will be forwarded to this service. You could update this so that only traffic to the _front-end_ path would be routed to this service by updating the path to be `path: 'front-end'`.
 
 There's also an optional health check path. This path is invoked every couple of seconds so that the load balancer can ensure your service is healthy. By default the health check path is `/` - but this can be changed to anything.
 
 #### Scaling
 
-The next sectionin includes the resources allocated to your service. Load Balanced Web Services are run on AWS Fargate, meaning all you have to do is say how much CPU and memory your service needs. This section also includes how many coppies of your service you want up and running. By default, only one copy of your service is spun up, but that number can be increased to handle more load.
+The next sectionin includes the resources allocated to your service. Load Balanced Web Services are run on AWS Fargate, meaning all you have to do is say how much CPU and memory your service needs. This section also includes how many copies of your service you want up and running. By default, only one copy of your service is spun up, but that number can be increased to handle more load.
```

Comment:
TODO: sectionin => section 

I thought I made this change, will do on the next one